<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>800</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1100</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Главный экран</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(26,11,20, 255), stop:1 rgba(0, 12, 40, 255));
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="openFolder_button">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>340</y>
      <width>250</width>
      <height>110</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(108, 100, 0, 255), stop:1 rgba(52, 0, 50, 255));
border-radius: 13px;
border: 2px solid;</string>
    </property>
    <property name="text">
     <string>Открыть Папку</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openPhoto_button">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>340</y>
      <width>250</width>
      <height>110</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(108, 100, 0, 255), stop:1 rgba(52, 0, 50, 255));
border-radius: 13px;
border: 2px solid;</string>
    </property>
    <property name="text">
     <string>Открыть фото</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_open_photo_in_bd">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>501</width>
      <height>110</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(108, 100, 0, 255), stop:1 rgba(52, 0, 50, 255));
border-radius: 13px;
border: 2px solid;</string>
    </property>
    <property name="text">
     <string>Открыть личные фото</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
